server/service/integration_core_test.go

```diff
@@ -8675,6 +8675,7 @@ func (s *integrationTestSuite) TestGetHostDiskEncryption() {
 	s.DoJSON("GET", fmt.Sprintf("/api/latest/fleet/hosts/%d", hostLin.ID), nil, http.StatusOK, &getHostResp)
 	require.Equal(t, hostLin.ID, getHostResp.Host.ID)
 	require.Nil(t, getHostResp.Host.DiskEncryptionEnabled)
+	require.Nil(t, getHostResp.Host.MDM.OSSettings)
```

Comment:
Is this testing the change because disk encryption is not set in the "No team" config?

Reply:
Since [this method](https://github.com/fleetdm/fleet/blob/6f3a7dd95b5609e272804b1f47222d35e86a27af/server/datastore/mysql/app_configs.go#L277-L290) checks the team and "no team" (app) config, it should be?